pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories.md

```diff
@@ -0,0 +1,182 @@
+---
+$title: Embed stories in web pages
+$order: 3
+description: 'The AMP Story Player enables you to embed a story that users are able to tap or click through, inside of a web page. Follow this step-by-step guide to learn how.'
```

Comment:
FYI, the PRs that enable support for several stories and tap-to-next story are now in production (and swipe-to-next in is beta). So we could update this and following references to "embed stories".

Reply:
Good call out - I'll fix this with the next push